test/bigtcp/test.sh

```diff
@@ -43,7 +43,7 @@ helm install cilium ${HELM_CHART_DIR} \
 kubectl -n kube-system rollout status ds/cilium --timeout=5m
 
 # check if BIG TCP is initialized
-kubectl -n kube-system logs ds/cilium 2>&1 | grep "Setting up IPv6 BIG TCP"
+kubectl -n kube-system logs ds/cilium 2>&1 | grep "Setting up BIG TCP"
```

Comment:
thanks for the fix. this could probably be submitted as a separate PR and we can already merge it.

Reply:
Done: https://github.com/cilium/cilium/pull/26377